cardano_node_tests/utils/dbsync_queries.py

```diff
@@ -691,3 +691,13 @@ def query_datum(datum_hash: str) -> Generator[DatumDBRow, None, None]:
     with execute(query=query, vars=(rf"\x{datum_hash}",)) as cur:
         while (result := cur.fetchone()) is not None:
             yield DatumDBRow(*result)
+
+
+def query_cost_model() -> Any:
```

Comment:
should be probably `-> Dict[str, Any]`

Reply:
mypy doesn't allow it

Reply:
See https://github.com/input-output-hk/cardano-node-tests/pull/1563/commits/ac0841022a0b9bddb61e94b14cb9cd96289c170c